Packages/vcs/vcs/Canvas.py

```diff
@@ -4961,6 +4979,15 @@ def setcolormap(self, name):
         #   updateVCSsegments_flag = args[1]
         # except:
         #   updateVCSsegments_flag = 1
+        # Python < 3 DeprecationWarning ignored by default
```

Comment:
thanks. @doutriaux1 I am bit confused. Currently the warning is in "setcolormap" instead. Look at the code at line 4953.